src/loaddb/load_db.c

```diff
@@ -549,6 +549,10 @@ loaddb_internal (UTIL_FUNCTION_ARG * arg, int dba_mode)
   /* login */
   if (!args.user_name.empty () || !dba_mode)
     {
+      if (strcasecmp (args.user_name.c_str (), "DBA") == 0)
```

Comment:
Can't it check with dba_mode?

Reply:
dba_mode is only set when calling the loaddb_dba() function.
loaddb_dba () is set as an unused function.
So we can only check by username.